tracer/src/Datadog.Trace/CreateSpanSettings.cs

```diff
@@ -0,0 +1,33 @@
+﻿// <copyright file="CreateSpanSettings.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+namespace Datadog.Trace
+{
+    /// <summary>
+    /// Settings to use when creating a new <see cref="IScope"/> and <see cref="ISpan"/>
+    /// </summary>
+    public struct CreateSpanSettings
```

Comment:
I'm really bad at naming but I would have went with SpanCreationSettings

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/2131/commits/5a05d3d7c4f4bc9e05c1580758ec1585d3ed3c97